modules/core/client/components/NavigationLoggedIn.js

```diff
@@ -48,15 +48,15 @@ export default function NavigationLoggedIn({ currentPath, onSignout, user }) {
           id="support-dropdown"
           title={t('Support')}
         >
-          <DropMenuItem href="/faq">
+          {/*           <DropMenuItem href="/faq">
```

Comment:
Prefer delete vs. commenting so that code is smaller and active

Reply:
Is this a blocker? I'm not sure if we want to delete everything because we would like to add flags and commenting out stuff sounds like a good compromise, otherwise we end up with big merge conflicts in the future or it makes it difficult to re-add the code?

Reply:
If you're really going to add it right back, then ok, but git and github make getting old code snippets pretty easy so something to keep in mind for future. Less live code, means easier maintenance long term.